frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/StudioCodeListEditor.test.tsx

```diff
@@ -344,6 +344,74 @@ describe('StudioCodeListEditor', () => {
     });
   });
 
+  describe('onCreateTextResource', () => {
+    it('Calls the onCreateTextResource callback with the new text resource when a help text is changed and there is no text resource linked to it', async () => {
```

Comment:
"Change" kan forveksles med `change`-hendelsen. Vi vil bare kalle denne når feltet mister fokus. Hva med en slik beskrivelse?
```suggestion
    it('Calls the onCreateTextResource callback with the new text resource when a help text field loses focus and there is no text resource linked to it', async () => {
```
Dette gjelder alle de nye testene her.